src/integration-tests/introspector/introspectTest.sh

```diff
@@ -742,6 +742,128 @@ function checkFileStores() {
   fi
 }
 
+#############################################################################
+#
+# Validate NODEMGR_MEM_ARGS environment variable values (-Xms64m -Xmx100m)
+# applied to Node Manager command line.
+#
+
+function checkNodeManagerMemArg() {
+
+  trace "Verifying node manager memory arguments"
+
+  # Verify that NODEMGR_MEM_ARGS environment value was applied to the Node Manager command line
+  linecount="`kubectl exec -it -n ${NAMESPACE} ${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE?}1 \
+       grep "\-Xms64m -Xmx100m" /shared/logs/${MANAGED_SERVER_NAME_BASE?}1_nodemanager.out \
+       | grep -v "NODEMGR_MEM_ARGS"  | wc -l`"
+  logstatus=0
+
+  if [ "$linecount" != "1" ]; then
+    trace "Error: The latest log from 'kubectl -n ${NAMESPACE} logs ${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE?}1' does not contain exactly 1 line that match ' grep '-Xms64m -Xmx100m' ', this probably means that it's reporting NODEMGR_MEM_ARGS not applied"
+    logstatus=1
+  fi
+
+  if [ $logstatus -ne 0 ]; then
+    exit 1
+  fi
+
+  # Verify that USER_MEM_ARGS environment value did not get applied to the Node Manager command line
+  maxRamlinecount="`kubectl exec -it -n ${NAMESPACE} ${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE?}1 \
+       grep "MaxRAMFraction=1" /shared/logs/${MANAGED_SERVER_NAME_BASE?}1_nodemanager.out \
+       | grep -v "JAVA_OPTIONS" | wc -l`"
+  logstatus=0
+
+  if [ "$maxRamlinecount" != "0" ]; then
+    trace "Error: The latest log from 'kubectl -n ${NAMESPACE} logs ${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE?}1' does not contain exactly 0 lines that match ' grep 'MaxRAMFraction=1' ', this probably means that it's reporting USER_MEM_ARGS was applied"
+    logstatus=1
+  fi
+
+  if [ $logstatus -ne 0 ]; then
+    exit 1
+  fi
+}
+
+#############################################################################
+#
+# Validate USER_MEM_ARGS environment variable values (-MaxRAMFraction)
+# applied to Managed Server command line.
+#
+function checkManagedServer1MemArg() {
+
+  trace "Verifying managed server memory arguments"
+
+  # Verify that USER_MEM_ARGS environment value was applied to the Managed Server 1 command line
+  maxRamlinecount="`kubectl exec -it -n ${NAMESPACE} ${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE?}1 \
+       grep "MaxRAMFraction=1"  /shared/logs/${MANAGED_SERVER_NAME_BASE?}1.out \
+       | grep -v "JAVA_OPTIONS" | wc -l`"
+  logstatus=0
+
+  if [ "$maxRamlinecount" != "1" ]; then
+    trace "Error: The latest log from 'kubectl -n ${NAMESPACE} logs ${DOMAIN_UID}-${MANAGED_SERVER_NAME_BASE?}1' does not contain exactly 1 line that match ' grep 'MaxRAMFraction=1' ', this probably means that it's reporting USER_MEM_ARGS not applied"
+    logstatus=1
+  fi
+
+  if [ $logstatus -ne 0 ]; then
+    exit 1
+  fi
+
+  # Verify that USER_MEM_ARGS environment value did not get applied to the Node Manager command line
```

Comment:
should comment be "Verify that NODEMGR_USER_MEM_ARGS environment value did not get applied to the Managed Server 1 command line"?